src/app/core/components/top-right-controls/top-right-controls.component.html

```diff
@@ -1,15 +1,21 @@
 <div class="controls-zone" [ngClass]="drawLeftBorder ? 'with-left-border' : 'no-left-border'" *ngrxLet="session$ as session">
-  <alg-language-picker *ngIf="!session || (session && session.user.tempUser)"></alg-language-picker>
-  <alg-top-right-menu *ngIf="session && !session.user.tempUser; else notConnected"></alg-top-right-menu>
-  <ng-template #notConnected>
+
+  <ng-container
+    *ngIf="session && !session.user.tempUser; then authenticated else notAuthenticated"
+  ></ng-container>
+
+  <ng-template #authenticated>
+    <alg-top-right-menu></alg-top-right-menu>
+  </ng-template>
+
+  <ng-template #notAuthenticated>
+    <alg-language-picker></alg-language-picker>
     <p-button
-      *ngIf="!session || (session && session.user.tempUser)"
       i18n-label label="Log in / Sign in"
       icon="pi pi-power-off"
       iconPos="right"
       styleClass="login-button"
       (click)="login()"
     ></p-button>
   </ng-template>
```

Comment:
This change is not in the scope of this PR, so I'll detail it shortly:
The same condition `!session || !session.user.tempUser` was repeated thrice, I gathered them in one and split the view into 2 template.